_episodes/07-thresholding.md

```diff
@@ -126,87 +124,45 @@ Now is where the main work of the program takes place.
 
 ~~~
 # blur and grayscale before thresholding
-blur = cv2.cvtColor(src = image, code = cv2.COLOR_BGR2GRAY)
-blur = cv2.GaussianBlur(src = blur, 
-    ksize = (k, k), 
-    sigmaX = 0)
+blur = skimage.color.rgb2gray(image)
```

Comment:
this will transform the range to `(0.0, 1.0)` and change the dtype to `float`. I don't know what's best here, to rescale before the gaussian blur, or to rescale all threshold values mentioned in the text.